src/lib.rs

```diff
@@ -667,6 +669,33 @@ decl_module! {
 
             Ok(())
         }
+
+        #[weight = FunctionOf(
+            |args: (&T::AccountId, &u8, &[u8; 32], &Timepoint<T::BlockNumber>)| 500_000 * (*args.1 as u32),
+            DispatchClass::Normal,
+            true
+        )]
+		fn clear(origin, multi_account_id: T::AccountId, max_cancellations: u8, call_hash: [u8; 32], timepoint: Timepoint<T::BlockNumber>) -> DispatchResult {
+			let mut succeeded = 0;
+			let who = ensure_signed(origin)?;
+			let mut cancelled_multisigs = Vec::with_capacity(max_cancellations as usize);
+			let active_multisigs = <Multisigs<T>>::iter_prefix(&who);
+			for m_sig in active_multisigs.take(max_cancellations as usize){
+				ensure!(m_sig.when == timepoint, Error::<T>::WrongTimepoint);
```

Comment:
Since multisigs are created in different timepoints, we cannot have this check here, I dont think we need it.